app/src/main/java/net/squanchy/search/SearchActivity.kt

```diff
@@ -46,13 +46,16 @@ class SearchActivity : AppCompatActivity(), SearchRecyclerView.OnSearchResultCli
     private lateinit var searchTextWatcher: SearchTextWatcher
 
     private var hasQuery: Boolean = false
+    private lateinit var query: String
```

Comment:
Maybe `pendingQuery`? Or `initialQuery`?

Reply:
fixed in ac9e687